src/style.css

```diff
@@ -1 +1,63 @@
 /* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+}
+
+.header__wrapper {
```

Comment:
the wrapper is redundant
apply these styles to `.header` (add the class `header` to header tag)